packages/look-and-feel/css/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("../config/eslint.config.js");
```

Comment:
Si on garde cette structure de dossier ici il faudrait mettre
```suggestion
module.exports = require("../../config/eslint.config.js");
```